nfc.py

```diff
@@ -2,7 +2,9 @@
 
 import logging
 import sys
-import usb.core
+import pkg_resources
+import subprocess
+#import usb.core 	imported later on in order to allow checking weather it is installed
```

Comment:
Please import it in this place - if you want to detect whether module exits, catch [ImportError](https://docs.python.org/3/library/exceptions.html#ImportError) exception here!